flatpak_builder_lint/appstream.py

```diff
@@ -12,8 +12,33 @@
 def validate(path: str) -> dict:
     if not os.path.isfile(path):
         raise FileNotFoundError("AppStream file not found")
+
+    overrides = {
+        "all-categories-ignored": "error",
+        "category-invalid": "error",
+        "cid-desktopapp-is-not-rdns": "error",
+        "cid-has-number-prefix": "error",
+        "cid-maybe-not-rdns": "error",
+        "cid-missing-affiliation-gnome": "error",
+        "cid-rnds-contains-hyphen": "error",
```

Comment:
Does this exist? I don't see it anywhere https://github.com/search?q=repo%3Aximion%2Fappstream+cid-rnds-contains-hyphen&type=code